INSTALL

```diff
@@ -14,24 +14,13 @@ The source for the file utility + library is available from
     ftp://ftp.astron.com/pub/file/
 
 You will need a cryptographic library to support digests and signatures.
-This library may be libgcrypt, Mozilla NSS, OpenSSL or beecrypt.
+This library may be libgcrypt, Mozilla NSS or OpenSSL.
 Which library to use can be specified with the
---with-crypto=[libgcrypt|beecrypt|nss|openssl] argument to configure,
+--with-crypto=[libgcrypt|nss|openssl] argument to configure,
```

Comment:
Shouldn't be the NSS references removed similarly to beecrypt?

Reply:
Oh, of course. Thanks for spotting!